single.mustache

```diff
@@ -15,6 +15,7 @@ URL: {{{PROJECTURL}}}
 {{/each}}
 %if 0%{?!scl:1}
 BuildRequires: nodejs-packaging
+BuildRequires: npm
```

Comment:
Why is NPM needed? Does that provide the packaging macros on EL8? The only reason we install it is for the `%{nodejs_sitelib}` macro and other packaging scripts. NPM itself isn't ran.

Reply:
It failed trying to call npm on the el8 builds, not sure why :-/

Reply:
Do you have a link to a failed task?